nginx/tests/test_unit.py

```diff
@@ -39,14 +39,15 @@ def test_flatten_json_timestamp(check):
 def test_plus_api(check, instance, aggregator):
     instance = deepcopy(instance)
     instance['use_plus_api'] = True
+    instance['plus_api_version'] = 3
```

Comment:
I think it's better to duplicate the test

Reply:
Agreed